samples/meeting-av-filters/csharp/MeetingAVFilter/MeetingAVFilter.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Can't we use .Net 6

Reply:
This app mainly focuses on client app only with no dependency on .net framework so it will be fine